samples/cpp/benchmark_app/main.cpp

```diff
@@ -55,6 +55,11 @@ bool parse_and_check_command_line(int argc, char* argv[]) {
     if (FLAGS_api != "async" && FLAGS_api != "sync") {
         throw std::logic_error("Incorrect API. Please set -api option to `sync` or `async` value.");
     }
+    if (FLAGS_hint != "none" && (FLAGS_nstreams != "" || FLAGS_nthreads != 0 || FLAGS_pin != "")) {
```

Comment:
There is a sense to move this check below, after validation of `FLAGS_hint` value itself.

Reply:
I don't see much sense in that as these checks test different usecases. But I don't have objection, sequence is changed.